db/seed_maker.rb

```diff
@@ -336,8 +336,9 @@ def setup_hmis_admin_access
 
     # Create HMIS Data Source
     hmis_ds = GrdaWarehouse::DataSource.source.where(hmis: ENV['HMIS_HOSTNAME']).first_or_create! do |ds|
-      ds.name = 'HMIS Data Source'
+      ds.name = 'HMIS'
       ds.short_name = 'HMIS'
+      ds.authoritative = true
```

Comment:
Hmm, this is probably good, but I think we're going to need more code to prevent the ability to add clients directly to the authoritative data source.  We should limit the data sources available here: app/views/clients/_new_client_form.haml to those that are authoritative, and NOT HMIS.

Reply:
Oh, yeah. Records from the HMIS data source should not be editable _at all_ in the warehouse, right? I thought that was already achieved by the fact that warehouse deals with Destination clients only. 
I will look at this new_client feature, I'm not familiar with it.

Reply:
Actually this doesn't feel like it needs to take priority right now, but I added a ticket to the backlog https://www.pivotaltracker.com/story/show/185082586